src/main/webapp/index.html

```diff
@@ -15,46 +15,70 @@
     <header>Good News - Bad News</header>
 
     <div id = "main">
-      <aside>
+      <aside id="topLow">
 
 	<article id = "topGood">
+	  <h3>Top Good Today</h1>
 	  <figure>
 	    <img>
-	    <figcaption>topGood</figcaption>
+	    <figcaption></figcaption>
 	  </figure>
+	  <h6 id ="tgTittle"></h6>
+	  <p id = thSnip>this will be a snippet of post txt</p>
 	</article>
 
 	<article id = "lowBad">
+	  <h3>Low Bad Today</h3>
 	  <figure>
 	    <img>
-	    <figcaption>lowBad</figcaption>
+	    <figcaption></figcaption>
 	  </figure>
+	  <h6 id="lbTittle"></h6>
```

Comment:
This is the first occurrence but there's a bunch of places where Title has two "t"

Reply:
cool i got the titles right now and went back and i think the newest commit should fix the thing